.pre-commit-config.yaml

```
@@ -26,16 +26,11 @@ repos:
       # exclude generated files
       exclude: ^validation/|\.dtd$|\.xml$
 
--   repo: https://github.com/asottile/pyupgrade
-    rev: v3.3.1
-    hooks:
-    - id: pyupgrade
-      args: ["--py38-plus"]
-
--   repo: https://github.com/MarcoGorelli/absolufy-imports
-    rev: v0.3.1
-    hooks:
-    - id: absolufy-imports
+- repo: https://github.com/charliermarsh/ruff-pre-commit
+  rev: "v0.0.253"
```

```suggestion
  rev: "v0.0.255"
```

just bumping to latest version.